install/kubernetes/cilium/charts/config/templates/configmap.yaml

```
@@ -165,10 +165,14 @@ data:
   bpf-ct-global-tcp-max: "{{ .Values.global.bpf.ctTcpMax }}"
   bpf-ct-global-any-max: "{{ .Values.global.bpf.ctAnyMax }}"
 
-  # bpf-nat-global-max specified the maximum number of entries in the BPF NAT
-  # table.
+  # bpf-nat-global-max specified the maximum number of entries in the
```

Nit: could fix a typo here while at it

```suggestion
  # bpf-nat-global-max specifies the maximum number of entries in the
```